packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -34,15 +46,21 @@ class NodeMessageBase(ProjectMessageBase):
 
 
 class LoggerRabbitMessage(RabbitMessageBase, NodeMessageBase):
-    channel_name: Literal["simcore.services.logs"] = "simcore.services.logs"
+    channel_name: Literal["simcore.services.logs.v2"] = "simcore.services.logs.v2"
     messages: list[str]
     log_level: int = logging.INFO
```

Comment:
MINOR: we could define ``LogInt`` either with an [``IntEnum``](https://docs.python.org/3/library/enum.html#enum.IntEnum) or as a ``Literal[DEBUG, ...]`` of https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#LL98C1-L105C11